docker/docker-compose.yml

```diff
@@ -40,10 +40,6 @@ services:
       - model-cache:/cache
     env_file:
       - .env
-    environment:
-      - NODE_ENV=production
```

Comment:
we will need this one to toggle hot-reload in development

Reply:
This container doesn't use node anymore ;)

Reply:
we still use that environment variable name 😢

Reply:
Gotcha